docs/build/interoperate-qiskit-qasm2.mdx

```diff
@@ -116,28 +240,63 @@ import qiskit.qasm2
 program = '''
     include "qelib1.inc";
     qreg q[2];
-    rx(atan2(pi, 3 + add_one(0.2))) q[0];
+    rx(arctan(pi, 3 + add_one(0.2))) q[0];
     cx q[0], q[1];
 '''
 
 def add_one(x):
     return x + 1
 
 customs = [
-    # `atan2` takes two parameters, and `math.atan2` implements it.
-    qasm2.CustomClassical("atan2", 2, math.atan2),
     # Our `add_one` takes only one parameter.
     qasm2.CustomClassical("add_one", 1, add_one),
+    # `arctan` takes two parameters, and `math.atan2` implements it.
+    qasm2.CustomClassical("atan2", 2, math.atan2),
 ]
 circuit = qasm2.loads(program, custom_classical=customs)
 ```
 
 <span id="strict"></span>
-## Strict mode
+### Strict mode
+
+By default, this parser is more relaxed than the official specification.
+It allows trailing commas in parameter lists; unnecessary (empty-statement) semicolons; omission of the `OPENQASM 2.0;` version statement; and several other quality-of-life improvements without emitting any errors.
+However, you can use the "letter-of-the-spec" mode with `strict=True`.
+
+
+<span id="export"></span>
+## Export a Qiskit circuit to OpenQASM 2
+
+Qiskit can also export a [`QuantumCircuit`](../api/qiskit/qiskit.circuit.QuantumCircuit) to OpenQASM 2.
+You use the function [`qasm2.dump()`](../api/qiskit/qasm2#dump) to write to a file, and [`qasm2.dumps()`](../api/qiskit/qasm2#dumps) to write to a string.
+These functions currently have a very simple interface: they accept a circuit and, only in the case of [`qasm2.dump()`](../api/qiskit/qasm2#dump), a location to write the output to.
+
+<Admonition type="warning">
+Qiskit's OpenQASM 2 exporter still assumes a legacy, non-standard version of the `"qelib1.inc"` include file.
+[We are planning to fix this in a later release of Qiskit](https://github.com/Qiskit/qiskit/issues/10737), but in the mean time, if you need to re-import an OpenQASM 2 program created with Qiskit, you should use [the example above for how to tell the importer about the legacy gates](#qasm2-import-legacy).
```

Comment:
```suggestion
[This will be resolved in a later release of Qiskit](https://github.com/Qiskit/qiskit/issues/10737), but in the mean time, if you need to re-import an OpenQASM 2 program created with Qiskit, use [the example above for how to tell the importer about the legacy gates](#qasm2-import-legacy).
```